server/api/crud/artifacts.py

```diff
@@ -59,6 +66,23 @@ def store_artifact(
             producer_id=producer_id,
         )
 
+    @staticmethod
+    def _resolve_artifact_size(artifact, auth_info):
+        if "spec" in artifact and "size" not in artifact["spec"]:
+            if "target_path" in artifact["spec"]:
+                path = artifact["spec"].get("target_path")
+                try:
+                    file_stat = server.api.crud.Files().get_filestat(
+                        auth_info, path=path
+                    )
+                    artifact["spec"]["size"] = file_stat["size"]
+                except Exception as err:
+                    logger.debug(
+                        "Calculating artifact size failed",
```

Comment:
Just to align with the rest of our failure logs
```suggestion
                        "Failed calculating artifact size",
```